### DISCUSSION TITLE
How to apply after getting student pack?

### DISCUSSION THREAD

**Discussion by devikavarshney** [community/community#56969] [GitHub Education]
How to apply after getting student pack?

**Comment by samus-aran** [comment]
Hi @devikavarshney  👋

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.

To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.
